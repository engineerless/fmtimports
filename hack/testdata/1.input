package main

import (
	"fmt"
    foo "github.com/foo"
    k8sbar "k8s.io/bar"
    bar "github.com/bar"
    k8sfoo "k8s.io/foo"
)

import (
	bootstraptokenv1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/bootstraptoken/v1"
	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
	"k8s.io/kubernetes/cmd/kubeadm/app/constants"

	corev1 "k8s.io/api/core/v1"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	runtimeserializer "k8s.io/apimachinery/pkg/runtime/serializer"

	fuzz "github.com/google/gofuzz"
    fuzz "github.com/google/gofuzz"
    "os"
    "fmt"
)

import (
	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2"
	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3"

	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	"k8s.io/apimachinery/pkg/runtime"
	"k8s.io/apimachinery/pkg/runtime/schema"
	"k8s.io/apimachinery/pkg/runtime/serializer"
	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
)
