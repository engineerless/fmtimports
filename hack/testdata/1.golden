package main

import (
	"fmt"

	bar "github.com/bar"
	foo "github.com/foo"
	k8sbar "k8s.io/bar"
	k8sfoo "k8s.io/foo"
)
